tests/unit/lms/resources/_js_config_test.py

```diff
@@ -443,15 +479,23 @@ def context():
         instance=True,
         h_group=mock.create_autospec(HGroup, instance=True, spec_set=True),
         is_canvas=True,
+        should_use_section_groups=False,
     )
 
 
+@pytest.fixture
+def section_groups_on(context):
+    context.should_use_section_groups = True
+
+
 @pytest.fixture
 def pyramid_request(pyramid_request):
     pyramid_request.params["lis_result_sourcedid"] = "example_lis_result_sourcedid"
     pyramid_request.params[
         "lis_outcome_service_url"
     ] = "example_lis_outcome_service_url"
+    pyramid_request.params["context_id"] = "test_course_id"
+    pyramid_request.params["custom_canvas_course_id"] = "test_course_id"
```

Comment:
```suggestion
    pyramid_request.params.update(
        {
            "lis_result_sourcedid": "example_lis_result_sourcedid",
            "lis_outcome_service_url": "example_lis_outcome_service_url",
            "context_id": "test_course_id",
            "custom_canvas_course_id": "test_course_id",
        }
    )
```

Also is the one above duplicating "context_id" and "custom_canvas_course_id" or does it not inherit from this?

Reply:
:+1: The one above clears and then re-sets the params for clarity/independence

Reply:
Righto

Reply:
https://github.com/hypothesis/lms/pull/1680/commits/2cef2fcab4fe143404c2936265bc7579864f4a64